public/javascripts/songlistTable.js

```diff
@@ -41,7 +41,7 @@ var songlistTable = {
                                     ]),
                                     m('td.alignWithTitle', [
                                         m('a', {
-                                            href: '/song/' + s._id
+                                            href: '/song/' + s._id + '?left=' + s.lang.code +  '&right='
```

Comment:
if we dont specify right then it shows nothing on the right column. the reason why i initialize it with '&right=' is so we can just detect the word 'right=' and change the string after 'right=' in order to change what the right column displays. if we dont do that then we need to have two separate logics, if it is the first time user choose the picker on the right we want to append '&right=', if not then replace it.